swarm/tools/runs_gc.py

```
@@ -73,14 +74,27 @@ def age_days(self) -> float:
+            current_dir = dirs_to_scan.pop()
+            try:
+                with os.scandir(current_dir) as it:
+                    for entry in it:
+                        # Prevent following symlinks to avoid infinite loops
+                        if entry.is_dir(follow_symlinks=False):
+                            dirs_to_scan.append(entry.path)
+                        elif entry.is_file():
+                            try:
+                                total += entry.stat().st_size
+                            except OSError:
+                                pass
```

Ruff (0.15.6)

[warning] Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(OSError): ...`

(SIM105)